pkgs/development/interpreters/php/8.1.nix

```
@@ -4,6 +4,7 @@ let
   base = callPackage ./generic.nix (_args // {
     version = "8.1.27";
     hash = "sha256-oV/XPqRPLfMLB9JHhuB9GUiw6j7tC4uEVzXVANwov/E=";
+    extraPatches = [ ./libxml212.patch ];
```

Would be nice to document the provenance. Something like:

```suggestion
    extraPatches = [
      # Fix build with libxml 2.12+.
      # Rebased from https://github.com/php/php-src/commit/0a39890c967aa57225bb6bdf4821aff7a3a3c08
      ./libxml212.patch
    ];
```

---

Fixed